IFC4x3/Properties/t/TensileStrengthAfterStranded_3uq2zgfMj9p9RWZO_R4dLy/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TensileStrengthAfterStranded_3uq2zgfMj9p9RWZO_R4dLy" Name="TensileStrengthAfterStranded" UniqueId="f8d02f6a-a56b-49cc-96e0-8d8f9b12757c" PrimaryDataType="IfcForceMeasure" />
```

Comment:
Could be generalized as `TensileStrength`?

Reply:
See comment on mechanical properties